src/makeCalculator.js

```diff
@@ -37,7 +37,64 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const methods = {
+    result: 0,
+
+    add: (number) => {
+      methods.result += number;
+
+      return methods;
+    },
+
+    subtract: (number) => {
+      methods.result -= number;
+
+      return methods;
+    },
+
+    multiply: (number) => {
+      methods.result *= number;
+
+      return methods;
+    },
+
+    divide: (number) => {
+      if (number !== 0) {
```

Comment:
If the number is 0, then a message must be displayed that it is not possible to divide by 0